### PR TITLE
Draft proposal for a o11y index page

### PR DESCRIPTION
**Pull request by ArthurSens**
This is a draft proposal for #19 that I'd like to get some opinions on our next meeting.

What I'd like to discuss:
* What people think about this table-driven layout?

* Are we "collecting" materials about Observability only, or do we want project-specific as well? E.g., my questions at #18 were strongly related to Prometheus, do we want to provide the answer here or to leave it to the Prometheus team?

* How do we accept community contributions to this index? 
  * We probably would like to avoid pages that provide the same information
  * We probably would like to avoid pages with a non-inclusive language
  * Do we accept a new entry in which the contributor is not the author? E.g., me adding Richard's o11y 101 video.

We need a written guide with all the requirements to accept a new entry.

Signed-off-by: arthursens <arthursens2005@gmail.com>

### CODE REVIEW

**Comment by ArthurSens** [comment]
Added :slightly_smiling_face:

File: index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+| Title                         | Author(s)                            | Publication date | Category  |
+|-------------------------------|--------------------------------------|------------------|-----------|
+| [Observability Maturity Report](https://cloud-native.slack.com/files/UPSTWGCBH/F01ER3VJY9Y/observability-maturity-report-4-3-2020-1-1.pdf) | ClearPath Strategies & honeycomb.io  | Q1, 2020         | Survey    |
+| [Monitoring and Observability](https://copyconstruct.medium.com/monitoring-and-observability-8417d1952e1c)  | Cindy Sridharan                      | Sep 5, 2017      | Blog post |
+
+### Read time: 30 mins up to 8 hours
+
+| Title                         | Author(s)                            | Publication date | Category  |
+|-------------------------------|--------------------------------------|------------------|-----------|
+| [Prometheus Docs](https://prometheus.io/docs/prometheus/latest/getting_started/) | Prometheus Authors  | Not applicable         | Docummentation    |
```
Comment (roidelapluie) [inline index.md:31]:
```suggestion
| [Prometheus Docs](https://prometheus.io/docs/prometheus/latest/getting_started/) | Prometheus Authors  | Not applicable         | Documentation    |
```

**Comment by halcyondude** [comment]
Revisiting this, is this draft still active?  Have discussions stalled?

**Comment by alolita** [comment]
No response on this PR. Closing for now. Let's reopen if there is renewed interest.

**Pull request closed without merging**